### ISSUE TITLE
Bug when restarting using Thompson microphysics

### ISSUE THREAD

**Issue by bbrashers** [wrf-model/WRF#1147]
See [https://forum.mmm.ucar.edu/phpBB3/viewtopic.php?f=53&t=5622](url) for a bug posting. 

Proposed solution: close unit 63 after reading **freezeH2O.dat** in **phys/module_mp_thompson.F**:

[code]CALL wrf_message("ThompMP: read freezeH2O.dat instead of computing")
          OPEN(63,file="freezeH2O.dat",form="unformatted",err=1234)
          READ(63,err=1234)tpi_qrfz
          READ(63,err=1234)tni_qrfz
          READ(63,err=1234)tpg_qrfz
          READ(63,err=1234)tnr_qrfz
          READ(63,err=1234)tpi_qcfz
          READ(63,err=1234)tni_qcfz
          CLOSE(63)                            ! BAB proposed solution
          good = 1
[/code]

**Comment by davegill** [comment]
@bbrashers 
Bart,
Take a look at 2a59ab1e9  #985 "Fix conflict between obs nudging and Thompson microphysics"

Is your fix needed in addition to this?

**Comment by bbrashers** [comment]
Yes, those fix the problem. I checked in WRF-4.1.3, and the close(63) call was still missing in phys/module_mp_thompson.F, which seems like the proper place to fix it. Open a file, read it, then immediate close it – good coding practice.

Bart

**Comment by davegill** [comment]
@bbrashers 
Bart,
If this does fix your problem, would you please provide this information on your forum post:
```
Commit 2a59ab1e9  
PR #985 "Fix conflict between obs nudging and Thompson microphysics"
```
Thanks!

**Comment by davegill** [comment]
@bbrashers 
If this does fix your problem, please close this issue

**Comment by bbrashers** [comment]
Fixed by Commit 2a59ab1e9  
PR #985 "Fix conflict between obs nudging and Thompson microphysics"
